services/graph/pkg/service/v0/users.go

```diff
@@ -1014,3 +986,34 @@ func isValidUserType(userType string) bool {
 
 	return false
 }
+
+func (g Graph) searchOCMAcceptedUsers(ctx context.Context, odataReq *godata.GoDataRequest) ([]*libregraph.User, error) {
+	users := []*libregraph.User{}
+	if g.config.IncludeOCMSharees {
+		gwc, err := g.gatewaySelector.Next()
+		if err != nil {
+			return nil, errorcode.New(errorcode.GeneralException, err.Error())
+		}
+		term, err := identity.GetSearchValues(odataReq.Query)
+		if err != nil {
+			return nil, errorcode.New(errorcode.InvalidRequest, err.Error())
+		}
+
+		remoteUsersRes, err := gwc.FindAcceptedUsers(ctx, &invitepb.FindAcceptedUsersRequest{Filter: term})
+		if err != nil {
+			// TODO grpc FindAcceptedUsers call failed
+			return nil, errorcode.New(errorcode.GeneralException, err.Error())
+		}
+		if remoteUsersRes.Status.Code != cs3rpc.Code_CODE_OK {
```

Comment:
```suggestion
		if remoteUsersRes.GetStatus().GetCode() != cs3rpc.Code_CODE_OK {
```

please use getters. My IDE gets angry otherwise.